apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2.IntegrationTests/MutateRowsTest.cs

```diff
@@ -0,0 +1,230 @@
+        }
+
+        [Fact]
+        public async Task DeleteFromColumn()
+        {
+            var tableName = _fixture.TableName;
+            var client = _fixture.TableClient;
+            var rowKey = await _fixture.InsertRowAsync(tableName, version: new BigtableVersion(1));
+            client.MutateRow(
+                tableName,
+                rowKey,
+                Mutations.SetCell(
+                    BigtableFixture.ColumnFamily1,
+                    "column_name",
+                    "abcd",
+                    new BigtableVersion(2)));
+
+            var stream = client.MutateRows(
+                tableName,
+                Mutations.CreateEntry(
+                    rowKey,
+                    Mutations.DeleteFromColumn(
+                        BigtableFixture.ColumnFamily1,
+                        "row_exists",
+                        new BigtableVersionRange(1, 2))));
+
+            var entries = await stream.GetResponseEntries();
+            Assert.True(entries.All(e => e.Status.Code == (int)Code.Ok));
+
+            await BigtableAssert.HasSingleValueAsync(
+                client,
+                tableName,
+                rowKey,
+                BigtableFixture.ColumnFamily1,
+                "column_name",
+                "abcd",
+                new BigtableVersion(2));
+        }
+
+        [Fact]
+        public async Task DeleteFromFamily()
+        {
+            var tableName = _fixture.TableName;
+            var client = _fixture.TableClient;
+            var rowKey = await _fixture.InsertRowAsync(tableName);
+            client.MutateRow(
+                tableName,
+                rowKey,
+                Mutations.SetCell(
+                    BigtableFixture.OtherColumnFamily,
+                    "last_name",
+                    "Smith",
+                    new BigtableVersion(3)));
+
+            var stream = client.MutateRows(
+                tableName,
+                Mutations.CreateEntry(
+                    rowKey,
+                    Mutations.DeleteFromFamily(BigtableFixture.ColumnFamily1)));
+
+            var entries = await stream.GetResponseEntries();
+            Assert.True(entries.All(e => e.Status.Code == (int)Code.Ok));
+
+            await BigtableAssert.HasSingleValueAsync(
+                client,
+                tableName,
+                rowKey,
+                BigtableFixture.OtherColumnFamily,
+                "last_name",
+                "Smith",
+                new BigtableVersion(3));
+        }
+
+        [Fact]
+        public async Task DeleteFromRow()
+        {
+            var tableName = _fixture.TableName;
+            var client = _fixture.TableClient;
+            var rowKey = await _fixture.InsertRowAsync(tableName);
+
+            var stream = client.MutateRows(
+                tableName,
+                Mutations.CreateEntry(
+                    rowKey,
+                    Mutations.DeleteFromRow()));
```

Comment:
So is this effectively "clear (but don't delete) row" or is there some more subtlety, e.g. that you can apply a column filter over the top?

Reply:
I believe it is effectively "clear _and delete_ row". There is no way to filter it.